test/ui-automation/test/specs/vault-interaction-flow.js

```diff
@@ -85,5 +85,21 @@ describe("TrustBloc Wallet - Add/Rename/Delete Vault flow", () => {
         await wallet.validateUserInput(invalidVaultName, "Must use letters (A-Z) and/or numbers (1-9)" );
         await wallet.validateUserInput(validVaultName, "There's already a vault with that name. Try something else." );
     });
-    //TODO: Issue-1393 Add locale support and test scenario for vault name with the spaces. 
+    it(`User enter vault name with spaces`, async function () {
+        // vault name with trailing spaces
+        await wallet.vaultNameWithSpaces('   Testing Vault  with spaces ', 'Testing Vault with spaces');
```

Comment:
You should just use multiple trailing spaces and no leading/repetitive spaces in this case (
`some vault name   `)